submissions/YuraZagor/Frogger/app.js

```diff
@@ -0,0 +1,108 @@
+const allEnemies = [];
+const speedLow = 200;
+const speedMedium = 275;
+const speedHigh = 350;
+const allSpeeds = [speedLow, speedMedium, speedHigh];
+const enemyRestartX = -55;
+const collisionOffset = 40
+const cellX = 101;
+const cellY = 83;
+const spriteRepositioningY = -10
+const playerStartX = 2*cellX;
+const playerStartY = 5*cellY + spriteRepositioningY ;
+const fieldWidth = 5*cellX;
+const fieldHight = 5*cellY;
+const row1 = cellY + spriteRepositioningY
+const row2 = 2*cellY + spriteRepositioningY
+const row3 = 3*cellY + spriteRepositioningY
+const rangesY = [row1, row2, row3];
```

Comment:
How about `const rangesY = [1, 2, 3].map(some maths here)`?
Whenever you are tempted to write `var1, var2, var3` it is a clear sign that you need to create an array with some maths.

Reply:
this is really great and concise, never thought that way. Done